src/gridfs/upload.ts

```diff
@@ -458,8 +458,8 @@ function getWriteOptions(stream: GridFSBucketWriteStream): WriteConcernOptions {
   if (stream.writeConcern) {
     obj.writeConcern = {
       w: stream.writeConcern.w,
-      wtimeout: stream.writeConcern.wtimeout,
-      j: stream.writeConcern.j
+      wtimeout: stream.writeConcern.wtimeoutMS,
```

Comment:
I think we should take the fix here a bit further, we're passing the legacy options into our own APIs here. The stream holds a WriteConcern instance already, I don't think we need any reshaping here, just pass it along

Reply:
Updated to pass through. The `getWriteOptions` method is now removed.